src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -514,40 +514,37 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
       return configResult.datasources[0].activeProvider!
     }
 
-    executeRaw(stringOrTemplateStringsArray, ...values) {
+    executeRaw(string: ReadonlyArray<string>, ...values: sqlTemplateTag.RawValue[]) {
       console.warn(
         `${chalk.yellow(
           'warn',
         )} prisma.executeRaw() is deprecated, please use prisma.$executeRaw() instead`,
       )
-      return this.$executeRaw(stringOrTemplateStringsArray, ...values)
+      return this.$executeRaw(string, ...values)
     }
 
     /**
      * Executes a raw query. Always returns a number
      */
-    private async $executeRawInternal(stringOrTemplateStringsArray, ...values) {
+    private async $executeRawInternal(string: ReadonlyArray<string>, ...values: sqlTemplateTag.RawValue[]) {
```

Comment:
As `sql-template-tag` now expects a `ReadonlyArray<string>` does it make sense to have the logic to check `isArray` or `typeof string`

Reply:
https://github.com/prisma/prisma/pull/3457/files/016f9c0eb50a684f004c99cb0d46aea35765e9bd#diff-fa9a06cd1e0a72798879057fc84c392b66c4cd06446619c74bc749f11aa152a1R529-R561